### PR TITLE
pt_PT templates

### PR DESCRIPTION
**Pull request by ambs**

### CODE REVIEW

File: plugins/generic/lucene/locale/pt_PT/emailTemplates.xml [hunk 1]
```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE email_texts SYSTEM "../../../../../lib/pkp/dtd/emailTemplateData.dtd">
```
Comment (asmecher) [inline plugins/generic/lucene/locale/pt_PT/emailTemplates.xml]: Too many `../`s here too.

Comment (ambs) [reply plugins/generic/lucene/locale/pt_PT/emailTemplates.xml]: From the US template `<!DOCTYPE email_texts SYSTEM "../../../../../lib/pkp/dtd/emailTemplateData.dtd">`

Comment (ambs) [reply plugins/generic/lucene/locale/pt_PT/emailTemplates.xml]: again, I might be sleeping

Comment (asmecher) [reply plugins/generic/lucene/locale/pt_PT/emailTemplates.xml]: Nope, you're absolutely right! Need more tea.

**Comment by asmecher** [comment]
Merged and cherry-picked to `ojs-stable-3_1_1`, thanks!

**Pull request merged**